tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.html

```diff
@@ -6,7 +6,7 @@
 	
 	<div class="gb-title">
 		<span wicket:id="externalAppFlag"></span>
-		<a wicket:id="title">Assignment title</a>
+		<a wicket:id="title"><wicket:container wicket:id="label" /></a>
```

Comment:
I should use this container tag more, so used to just sticking it on span's. I accidentally broke this with the addition of the link, and staring at the UI for so long the 'Assignment title' looked just fine! Good one.